packages/ui/src/composables/useSelectable.ts

```diff
@@ -100,7 +100,7 @@ export const useSelectable = (
       if (!props.modelValue) {
         emit('update:modelValue', [props.arrayValue])
       } else if (!Array.isArray(props.modelValue)) {
-        return
+        emit('update:modelValue', props.modelValue === props.arrayValue ? [] : [props.modelValue, props.arrayValue])
```

Comment:
I can't find arrayValue in our docs. What is it?

Reply:
https://vuestic.dev/en/ui-elements/checkbox
https://vuestic.dev/en/ui-elements/switch
This property is used to group input fields (`VaCheckbox`, `VaSwitch`)